src/Common/LoggingFormatStringHelpers.h

```diff
@@ -71,7 +71,7 @@ template <typename T> constexpr std::string_view tryGetStaticFormatString(T && x
     ///  - Use Exception::createRuntime or fmt::runtime if there's no format string
     ///    and a message is generated in runtime by a third-party library
     ///    or deserialized from somewhere.
-    static_assert(!std::is_same_v<std::string, std::decay_t<T>>);
+    // static_assert(!std::is_same_v<std::string, std::decay_t<T>>);
```

Comment:
This is odd and should not be merged in this state (I guess some of your code did not compiled with it?)